dds_web/api/schemas/user_schemas.py

```diff
@@ -4,6 +4,10 @@
 # IMPORTS ################################################################################ IMPORTS #
 ####################################################################################################
 
+# standar library
+import datetime
+from datetime import timedelta
```

Comment:
This should be possible to remove since we're anyway using `datetime.timedelta` and we're importing `datetime` above. 
```suggestion
```